CodeforAustralia/council-emissions-calculator-spike
# is217: fix responsiveness of table in `HowWeCalculate` (HWC) page

fixes #217 

## Overview of the Pull Request:
This PR fixes the issue reported in #217 
The table in HWC page has been update so that the entire table will fit within a mobile viewport without scroll left/right. see screenshot:

![image](https://user-images.githubusercontent.com/4408259/223716913-a74d7df9-55e2-46f1-a401-fc1cccc3108f.png)


## link to Trello card or Github issue
https://trello.com/c/VFI0iayC

## How Has This Been Tested?
* Check in the preview deployment that table in HWC page fits within mobile viewport (see above screenshot)
* Check in the preview deployment that table in HWC page renders correctly in desktop view

Screenshot of table in desktop view:
![image](https://user-images.githubusercontent.com/4408259/223718233-a30f5433-84dc-4678-ad09-55eae5e9bd4d.png)


## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!
